components/camel-mail/src/test/java/org/apache/camel/dataformat/mime/multipart/MimeMultipartDataFormatTest.java

```diff
@@ -261,9 +257,10 @@ public void marhsalOnlyMixed() throws IOException {
         headers.put("X-AdditionalData", "additional data");
         addAttachment("application/octet-stream", "foobar", "attachment.bin", headers);
         Exchange result = template.send("direct:marshalonlymixed", exchange);
-        assertThat(result.getOut().getHeader("Content-Type", String.class), startsWith("multipart/mixed"));
-        String resultBody = result.getOut().getBody(String.class);
-        assertThat(resultBody, containsString("Content-Description: Sample Attachment Data"));
+        assertTrue(result.getMessage().getHeader("Content-Type", String.class).startsWith("multipart/mixed"));
+        //assertThat(result.getMessage().getHeader("Content-Type", String.class), startsWith("multipart/mixed"));
```

Comment:
I would remove this comment

Reply:
yes. I forgot that one !!

Reply:
done !!

Reply:
Well done @zbendhiba, it's rebased and pushed upstream now :)